### DISCUSSION TITLE
子应用请求接口返回异常

### DISCUSSION THREAD

**Discussion by llbiao** [jd-opensource/micro-app#60] [Ideas]
![image](https://user-images.githubusercontent.com/32304163/130740784-b3ebc72d-8938-4341-bf54-7246d8a7a4ed.png)
![image](https://user-images.githubusercontent.com/32304163/130740864-0f611898-7a3b-4d04-893a-b960a162fff7.png)

**Comment by bailicangdu** [comment]
参考文档：https://zeroing.jd.com/micro-app/docs.html#/zh-cn/questions?id=_15%e3%80%81%e5%ad%90%e5%ba%94%e7%94%a8%e8%af%b7%e6%b1%82%e6%8e%a5%e5%8f%a3%e6%8a%a5%e9%94%99
